src/software/simulated_tests/kicker_double_touch.py

```diff
@@ -60,7 +60,17 @@ def get_validation_geometry(self, world) -> ValidationGeometry:
 
         """
         return create_validation_geometry(
-            [tbots_cpp.Field.createSSLDivisionBField().fieldLines()]
+            # [tbots_cpp.Field.createSSLDivisionBField().fieldLines()]
+            [
+                tbots_cpp.Circle(
+                    tbots_cpp.Point(
+                        robot.current_state.global_position.x_meters,
+                        robot.current_state.global_position.y_meters,
+                    ),
+                    0.5,
+                )
+                for robot in list(world.friendly_team.team_robots)
+            ]
```

Comment:
To improve this further, i think we should only visualize a circle around the kicker instead of all robots. 
```suggestion
            [
                tbots_cpp.Circle(
                    tbots_cpp.createPoint(self.kicker_robot.current_state.global_position),
                    0.2,
                )
            ] if self.kicker_robot is not None else []
```

Reply:
Playing the tests with this, it seems like `self.kicker_robot` is `None` for most of the test, even when there is a robot kicking the ball. Is that expected?

Reply:
not expected, I'm not sure why it does that but I'm trying to fix it

Reply:
I changed the process of choosing `self.kicker_robot`: previously it used `tbots_cpp.Robot(robot).isNearDribbler(ball_position, self.threshold)` with a threshold value of `0.5`, now it assigns `self.kicker_robot` to the robot closest in proximity to the ball at the start of play.

Reply:
I expected `0.5` to be a reasonable threshold considering the kicker robot starts at ~`0.3`m away from the ball but for some reason it doesn't work